src/createServer.js

```diff
@@ -1,3 +1,34 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { parseUrl } = require('./helpers/parseUrl.js');
+const { genereteResponse } = require('./helpers/generateResponse.js');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    if (req.url === '/favicon.ico') {
+      res.writeHead(204, { 'Content-Type': 'image/x-icon' });
+      res.end();
+
+      return;
+    }
```

Comment:
Didn't get why are you using here this piece of code for images

Reply:
I tested this server using browser, so create this piece of code to ignore favicon request